internal/controller/kyma/controller.go

```diff
@@ -673,3 +675,13 @@ func needUpdateForMandatoryModuleLabel(moduleTemplate v1beta2.ModuleTemplate) bo
 
 	return false
 }
+
+func setModuleStatusesToError(kyma *v1beta2.Kyma, optMessage string) {
```

Comment:
`opt` in `optMessage` indicating that it is optional? I am okay with it but have not seen this convention in the repo before... For me it would also be okay to just go with `message` instead.

Reply:
Yes. It constantly makes me uncomfortable how we are unable to idiomatically express optional values in golang.
You are right about the convention though. Renamed.

Reply:
> It constantly makes me uncomfortable how we are unable to idiomatically express optional values in golang

Agree